test-runner/src/main/resources/dcos-application.conf

```diff
@@ -0,0 +1,28 @@
+#Provide full path to the mesos native library. For ubuntu its filename is libmesos.so, for mac its libmesos.dylib
+# mit.mesos.native.library.location = "/usr/local/lib/libmesos.so"
+
+#Used to copy the application jar file for cluster mode tests
+# mit.hdfs.uri = ""
+
+#The IP address of the host as seen from docker instance. This ip address is used to connect to runner
+#from docker instance.
+#mit.docker.host.ip = ""
+
+#path of mesos slaves to find spark executor. This should be set when you create docker based mesos cluster
+# mit.spark.executor.tgz.location = "/spark"
+
+# mit.spark.mesos.executor.home = "/spark"
+
+# mit.spark.driver.host = "192.168.33.1"
+
+# mit.mesos.dispatcher.port = 7088
+
+mit.test.timeout = 5 minutes
+
+# mit.test.runner.port = 8888
+
+# SPARK_LOCAL_IP used to run spark-submit.sh
+# mit.spark_env.spark_local_ip = "192.168.33.1"
+
+# LIBPROCESS_IP used to run spark-submit.sh
+# mit.submit_env.libprocess_ip = "192.168.33.1"
```

Comment:
Since in sbt there is a difference between mit and dcos as tasks then "mit" prefix is misleading. Someone would expect dcos configs only here or common ones. For simplicity i would remove the mit prefix and use it only when it makes sense eg in the mit config file only. The same applies if a dcos specific configuration property is needed.